.github/workflows/publish_docker_images.yaml

```diff
@@ -24,14 +24,32 @@ jobs:
 
             - name: Build images
               run: |
-                  # Strip git ref prefix from version
-                  VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+                  DOCKER_TAGS=""
 
-                  # Strip "v" prefix from tag name
-                  [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+                  # Pull tag and branch from github.ref, which is either refs/heads/... or refs/tags/...
+                  TAG=$(echo "${{ github.ref }}" | sed -e '/refs\/tags\//!d; s,refs/.*/v\?\(.*\),\1,')
+                  BRANCH=$(echo "${{ github.ref }}" | sed -e '/refs\/heads\//!d; s,refs/.*/\(.*\),\1,')
```

Comment:
As before, but this time just outputs the branch name if the ref starts with `refs/heads/`